app/controllers/comments_controller.rb

```diff
@@ -0,0 +1,44 @@
+class CommentsController < ApplicationController
+  # def create
+  #   @comment = current_user.comments.new(comment_params)
+
+  #   respond_to do |format|
+  #     if @comment.save
+  #       format.html { redirect_to user_posts_path(current_user) }
+  #     else
+  #       format.html { render :show }
+  #     end
+  #   end
+  # end
+
+  # private
+
+  # def comment_params
+  #   params.require(:comment).permit(:text)
+  # end
+
+  def new
```

Comment:
- [ ] I couldn't create a comment, after creating a post I didn't see a form for creating a comment. I kindly advise you fix this as well.